easybuild/easyconfigs/c/CrayPGI/CrayPGI-19.06.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'CrayToolchain'
+
+name = 'CrayPGI'
+version = '19.06'
+
+homepage = 'https://pubs.cray.com/discover'
+description = """Toolchain using Cray compiler wrapper, PrgEnv-pgi compiler
+(PE release: June 2019).\n"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
```

Comment:
```suggestion
toolchain = {'name': 'dummy', 'version': ''}
```

Reply:
In general we remove the `version: dummy` and leave the `version` field empty

Reply:
we do that for regular easyconfigs, but I'm not sure if it has any side effects for toolchains. 

Maybe @boegel can help us here?

Reply:
It does matter, only when the `version` is empty for a `dummy` toolchain will the dependencies be loaded during installation time, see also https://easybuild.readthedocs.io/en/latest/Writing_easyconfig_files.html#loading-of-modules-for-dependencies-with-a-dummy-toolchain.

Since a toolchain is just a bundle of things for which a module file is generated (no actual compilation or anything like that), it doesn't matter here.

Note that we're getting rid of this in upcoming EasyBuild 4.0: we'll deprecate the use of `dummy` (`eb` will produce a warning if you use it, but it'll still work), and recommend switching to using the next `system` toolchain instead (only supported from EasyBuild 4.0 onwards), which is a drop-in replacement except for this funky behavior (the `system` toolchain totally ignores its `version`).
See also https://github.com/easybuilders/easybuild-framework/pull/2877

Reply:
Thanks @boegel